.gitignore

```diff
@@ -26,3 +26,4 @@ roachprod
 *.exe
 *.test
 *.prof
+.idea/
```

Comment:
We have a loosely-enforced company policy of avoiding editor-specific .gitignore entries. Here's my user gitignore, for example:

```
$ cat ~/.config/git/ignore
.DS_Store
.vscode
.idea
.terraform
```